src/main/java/frc/robot/Constants.java

```diff
@@ -53,4 +53,7 @@ public class Constants {
   public static final double SHOOTER_SPIN_UP_TIME = 1;
   public static final double INTAKE_SHOOT_SPEED = 1;
   public static final double SHOOTER_SHOOT_TIME = 1;
+
+public static final double SHOOTER_FLUSH_SPEED = 1;
```

Comment:
Is this a voltage or a speed?
Should these be negative?

Reply:
Speed and the negative is handled in the flush() in the intakeShooter file, so you can flush forward and backward